src/main.rs

```diff
@@ -533,9 +534,7 @@ fn main() {
 			let source = current_song.read_segment(&segment.id).buffered();
```

Comment:
Since `repeat_with_count` calls `buffered` we should probably remove it from here right?